src/models/member.model.ts

```diff
@@ -1,6 +1,12 @@
-export interface IMatchMembers {
+export interface IGetMyMember {
   userId: number;
   userName: string;
   profileImageUrl: string | null;
   isMatching: boolean;
+  localAddress: string | null;
+  coachingSports: {
+    sportId: number;
+    sportName: string;
+  }[];
```

Comment:
이것도 사실

```suggestion
  coachingSports: {
    id: number;
    name: string;
  }[];
```

이게 더 어울리는걸수도 ㅠㅠ